src/pages/Reservation/ReservUserInfo/ReservUserInfo.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import PassengerInfo from './PassengerInfo/PassengerInfo';
+import styled from 'styled-components';
+import Input from '../components/Input';
+import InputWrapper from '../components/InputWrapper';
+
+export default function ReservUserInfo() {
+  const [bookerInfo, setBookerInfo] = useState({
+    name: '',
+    email: '',
+    phonenumber: '',
+  });
+
+  const validateName = bookerInfo.name.length > 1 || bookerInfo.name === '';
+  const validateEmail =
+    bookerInfo.email.includes('@' && '.com') || bookerInfo.email === '';
+  const validateNumber =
+    bookerInfo.phonenumber.toString().length > 9 ||
+    bookerInfo.phonenumber === '';
+
```

Comment:
```suggestion
  const validator: {
    name: ...,
    email: ...,
    number: ...
  }
```
같은 뉘앙스를 공유하고 있는 코드이므로 객체로 묶어주면 좋을 것 같습니다